<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AFSWWI9CYZ2H4A" failures="0" tests="4" name="com.developerscorner.provider.service.AuthServiceTest" time="0.086" errors="0" timestamp="2023-02-09T09:17:50 CST" skipped="0">
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldThrowUsernameNotFoundExceptionIfUserNotExists" time="0.015"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldAuthenticateAUser" time="0.048"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldThrowConflictExceptionIfUserExists" time="0.004"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldRegisterAUser" time="0.019"/>
  <system-out/>
</testsuite> <!-- com.developerscorner.provider.service.AuthServiceTest -->
